app/components/atoms/ValidationMessages.tsx

```diff
@@ -0,0 +1,16 @@
+import { chakra, FormErrorMessage } from '@chakra-ui/react'
+
+interface ValidationMessagesProps {
+  errors: Array<string>
+}
+const ValidationMessages = ({ errors }: ValidationMessagesProps) => (
+  <FormErrorMessage>
+    <chakra.ul listStyleType="none">
+      {errors.map((error, index) => (
+        <li key={index}>{error}</li>
+      ))}
+    </chakra.ul>
```

Comment:
Hmm, pas sur du rendu, ca va faire plusieurs lignes. 
```suggestion
    <Text>
      {errors.join(', ')}
    </Text>
```

Reply:
Moi de tout maniere je pense que il faut affiicher que une seule erreur

Reply:
Niveau UX je trouve ça mieux que l'utilisateur soit informé de toutes les erreurs, comme ça il n'a pas à soumettre plusieurs fois le formulaire.

Si on imagine la création d'un mot de passe, on affiche toutes les erreurs d'un coup, sinon ça oblige l'utilisateur à le changer pleins de fois